Rakefile

```diff
@@ -0,0 +1,19 @@
+task :generate do
+  require 'open-uri'
+  require 'fileutils'
+  content = open("https://raw.githubusercontent.com/fastlane/fastlane/master/Rakefile").read
+  content = content.split("desc").first # only the first lines are relevant
+  eval(content)
```

Comment:
It's risky to evaluate code that is downloaded blindly from the Internet. :sweat_smile: